infra/create-app/prod.tfvars

```diff
@@ -1,8 +1,11 @@
 domain_name = "octopus.ac"
 
-allocated_storage       = 50
-instance                = "db.m5.large"
-db_version              = "14.5"
-backup_retention_period = 35
+rds_allocated_storage                     = 50
+rds_max_allocated_storage                 = 100
+rds_instance                              = "db.m5.large"
+rds_db_version                            = "14.8"
+rds_backup_retention_period               = 1
```

Comment:
should be 35

Reply:
Yes, bad copy and paste job from me. Now fixed!